superset/utils/core.py

```diff
@@ -72,6 +73,8 @@
 
 JS_MAX_INTEGER = 9007199254740991   # Largest int Java Script can handle 2^53-1
 
+BAD_CHARACTER_LIST = ['drop', 'alter', '--', ';', 'add' , 'update', 'union']
```

Comment:
I'd say the name is bad given the namespace. We either need a `utils/sql.py` or the name here should be something like `RISKY_SQL_SUBSTRINGS`.